.github/workflows/task-coverage-orchestrator.yml

```diff
@@ -0,0 +1,116 @@
+name: Task - Orchestrator Coverage
+
+# This workflow runs tests and generates code coverage reports
+# for the Orchestrator component
+on:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      madara-binary-hash:
+        description: "Hash used to retrieve the artifact"
+        required: true
+        type: string
+    secrets:
+      ETHEREUM_SEPOLIA_BLAST_RPC:
+        required: true
+      RPC_FOR_SNOS:
+        required: true
+
+jobs:
+  orchestrator_coverage:
+    runs-on: karnot-arc-runner-set
+
+    services:
+      localstack:
+        image: localstack/localstack@sha256:763947722c6c8d33d5fbf7e8d52b4bddec5be35274a0998fdc6176d733375314
```

Comment:
A comment about where this came from would be helpful

Reply:
This is pre-existing [from](https://github.com/madara-alliance/madara/blob/f42827cecedcd47aceb3a84074981ee5a62e5f8b/.github/workflows/e2e-test.yml#L15) [master](https://github.com/madara-alliance/madara/blob/f42827cecedcd47aceb3a84074981ee5a62e5f8b/orchestrator/README.md?plain=1#L215). i have no idea about the rational of choosing this godly sha.